src/MyClassEight.java

```diff
@@ -0,0 +1,17 @@
+public class MyClassEight {
+    public static void main(String[] args) {
+        String cssLocators = "CSS селектори: #id, name, .class, tagname, [attribute=value] - вони ефективні для пошуку елементів на сторінці.";
+        cssLocators += "XPath локатори: //tagname[@attribute='value'], //tagname[contains(@attribute, 'value')], //tagname[text()='value'] - XPath локатори дозволяють шукати елементи за допомогою складніших шаблонів.";
+        cssLocators += "Локатори за текстом: LinkText, PartialLinkText - дозволяють знаходити елементи за текстом.";
+
+        String xpathLocators = "XPath локатори для веб-елементів на табі Actions GitHub:\n";
+        xpathLocators += "Локатор для кнопки \"New workflow\": //a[@href='/actions/new-workflow']\n";
```

Comment:
It's not needed to comment what locator means. I or anyone else have to understand that reading it

Reply:
Each locator should have its informative name

Reply:
A few locators. According to the task should be 15